src/edu/csus/ecs/pc2/core/imports/clics/CLICSAwardUtilities.java

```diff
@@ -150,7 +150,7 @@ public static void addGroupWinners(IInternalContest contest, Run[] runs, List<CL
                 }
             }
         } catch (JAXBException | IllegalContestState | IOException e) {
-            StaticLog.getLog().log(Level.WARNING, "Error while calculating group winners", e);
+            StaticLog.getLog().log(Level.WARNING, "Error while calculating group winners DSA", e);
```

Comment:
I don't understand why this log message has been changed to _add_ the word "DSA" to it.  The method in which this `catch` clause appears makes no direct reference to the `DefaultScoringAlgorithm` (if that's what "DSA" is supposed to be referring to), and in any case it's not going to make sense to reference the DSA in the log message.

Note that earlier I commented on a similarly inappropriate reference to DSA in a message, and it was subsequently removed.  I don't understand why it's been _added_ here.